pkg/reconciler/mtbroker/delivery_config.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mtbroker
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/configmap"
+	"sigs.k8s.io/yaml"
+
+	eventingduck "knative.dev/eventing/pkg/apis/duck/v1"
+)
+
+const (
+	InternalDeliveryConfigMapName        = "mt-broker-internal-delivery"
+	internalDeliveryConfigMapDeliveryKey = "delivery"
```

Comment:
Any reason for making the name public but the key private? Can't they both be private?